pkg/ccl/importccl/import_stmt_test.go

```diff
@@ -5417,6 +5394,51 @@ func TestImportPgDump(t *testing.T) {
 	})
 }
 
+func TestImportPgDumpIgnoredStmts(t *testing.T) {
```

Comment:
Should we also add tests for unimplemented but "supported" statements to document their interaction with the `ignore_unsupported` flag?

Reply:
Yep, I will add this as part of the second commit which will be ignoring:
- Statements which we silently ignored previously.
- Statements which we error'ed on before during the tree.Node evaluation, but now ignore if the setting is set.